code/game/turfs/open/glass.dm

```diff
@@ -23,9 +24,11 @@
 	. = ..()
 	AddElement(/datum/element/turf_z_transparency, TRUE)
 
+
 /turf/open/floor/glass/reinforced
 	name = "Reinforced glass floor"
 	desc = "Do jump on it, it can take it."
 	icon = 'icons/turf/floors/reinf_glass.dmi'
 	icon_state = "reinf_glass-0"
 	base_icon_state = "reinf_glass"
+	broken_states = list("reinf_glass-damaged1", "reinf_glass-damaged2", "reinf_glass-damaged3")
```

Comment:
And this into `And this into `/turf/open/floor/glass/reinforced/setup_broken_states()``